PhotoMap/Features/Map/ViewModels/MapViewModelType.swift

```diff
@@ -9,17 +9,18 @@ import MapKit
 import Combine
 
 protocol MapViewModelInput {
-    var categoryButtonSubject: PassthroughSubject<Void, Never> { get set }
-    var enableDiscoveryModeSubject: PassthroughSubject<Void, Never> { get set }
-    var navigationButtonSubject: PassthroughSubject<Void, Never> { get set }
-    var photoButtonSubject: PassthroughSubject<Void, Never> { get set }
+    var categoryButtonSubject: PassthroughSubject<UIControl, Never> { get }
+    var enableDiscoveryModeSubject: PassthroughSubject<GestureType, Never> { get }
+    var navigationButtonSubject: PassthroughSubject<UIControl, Never> { get }
+    var photoButtonSubject: PassthroughSubject<UIControl, Never> { get }
 }
 
 protocol MapViewModelOutput {
     var tabTitle: String { get }
     var isShowUserLocation: Bool { get }
     var region: MKCoordinateRegion? { get }
     var modeButtonCollor: UIColor { get }
+    var isFollowModeOn: Bool { get }
```

Comment:
Add tests for other outputs (tabTitle, isShowUserLocation, region, modeButtonCollor)

Reply:
fix 6f58eb2